<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="liste">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>60</width>
      <height>28</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="myPlayListe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Play List</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setting">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Setting</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addsound">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>82</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>82</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>‚ûï</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="researchSound">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>41</y>
      <width>781</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSound">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>540</y>
      <width>600</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>600</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>600</width>
      <height>21</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="prev">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>530</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>‚èÆ</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePlay">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>530</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>‚è∏</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>530</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>‚è≠</string>
    </property>
   </widget>
   <widget class="QSlider" name="volumeSlider">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>540</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="musicSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>771</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>626</x>
      <y>530</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00 / 00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="movePlayList">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>530</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>üîÅ</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
